.github/ISSUE_TEMPLATE/mismatch-report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Mismatch report'
+about: Template for reporting a new entry for mismatch database
+title: 'feat: new product mismatch [product name]'
+labels: enhancement
+assignees: ''
+
+---
+
+### Description
+
+Add a short description of the invalid vendor-product relation.
```

Comment:
Let's give them a really easy to fill out list of what we need and what they likely care about.

```yaml
  purls:
    - [purl identifier] (e.g. pkg:pypi/zstandard.  This will be in the format pkg:[package repository][product name])
  invalid_vendors:
    - [vendor that shouldn't be detected] (e.g. facebook)    
```

We might also ask for a CVE number (which we can use to find the vendor)

```
Not sure how to fill this out?  Give us the CVE number that's being incorrectly detected and we'll try to figure it out from there.
```